### ISSUE TITLE
handle type Service records

### ISSUE THREAD

**Issue by jmckenna** [iodepo/oih-ui#133]
- some partners have JSON-LD with type [Service](https://schema.org/Service)
- however, if those records are loaded into Solr and exposed on the front-end, the entire front-end breaks 
  - because none of the 7 types supported by the front-end, are of type `Service`
  - and because the records contain spatial, when a user clicks on `Spatial Search` the entire page will break (JS error / blank page) see screen capture below
 - for several years now I 'get around this' by changing the type through the Solr indexer to something that is supported in the front-end (such as changing type `SoftwareSoureCode` to type `CreativeWork` ie. issue #131 )
   - I *could* temporarily change the type `Service` records to display as type `Dataset`, but, that seems incorrect
  
What is the way forward?

![Screenshot 2024-05-07 163225](https://github.com/iodepo/oih-ui/assets/1611709/f76d0dd2-e95d-4a78-81f3-ad016ea8e976)

**Comment by jmckenna** [comment]
other related issues:
- https://github.com/iodepo/odis-arch/issues/258
- https://github.com/iodepo/odis-arch/issues/360
- https://github.com/iodepo/oih-ui/pull/112
- https://github.com/iodepo/oih-ui/pull/87

**Comment by pbuttigieg** [comment]
No.  the TOR of the contracts issued to both the back and front end include terms for adding arbitrary types to the UI .

if we can't add a service facet, these TOR are violated. this is trivial- it's just another type.
